workers/common/src/main/kotlin/common/Extensions.kt

```diff
@@ -36,6 +37,14 @@ import org.slf4j.LoggerFactory
 
 val logger: Logger = LoggerFactory.getLogger(ConfigManager::class.java)
 
+/**
```

Comment:
Can you please give an outline in the commit message for what this function will be used? Because so far it's not even a real `feat` if it's just "dead code" 😉

Reply:
Also moving this commit to directly before the commit that makes use of the new function could help.

Reply:
> Also moving this commit to directly before the commit that makes use of the new function could help.

It already is.